kratos/utilities/divide_tetrahedra_3d_4.h

```diff
@@ -182,6 +185,8 @@ class KRATOS_API(KRATOS_CORE) DivideTetrahedra3D4 : public DivideGeometry<TPoint
     ///@name Private Inquiry
     ///@{
 
+    bool NodeIsInterface(int NodeKey) const;
+
```

Comment:
This is admittedly a bit picky, but I'd add this method to the triangle divider as well (it makes the code more self-comprehensive).